cyber-resilience-sig/README.md

```diff
@@ -6,9 +6,71 @@ Cyber Resilience Special Interest Group (SIG) of ORC WG.
 * Dirk-Willem van Gulik ([@dirkx][]), The Apache Software Foundation
 * Timo Perala ([@timop62][]), Nokia
 
-## Deliberables Plan
+## Scope
+
+The Scope of the Cyber Resilience SIG is a strict subset of the scope of ORC WG. Whereas the working group is chartered to address any kind of emerging regulation impacting open source, the Cyber Resilience SIG is solely focused on cyber resilience regulation. Expect all of the CRA-related work to happen in this SIG.
+
+In 2025, the SIG will focus on deliverables necessary to help the open source community (and notably _open source software stewards_) meet the regulatory obligations outlined in the CRA and help downstream users (_manufacturers_) be able to continue to leverage open source in their products and services while meeting their own regulatory requirements. This is expressed by the colored nodes and edges in the diagram below.
+
+```mermaid
+flowchart LR
+    foss["Open source project"] == integrated by ==> manufacturer["Manufacturer"] 
+    manufacturer == performs due diligence ==> foss
+    steward["Steward"] == reports vulnerabilities & incidents to ===> enisa["ENISA / CSIRT"]
+    steward == colaborates with ===> msa["Market Surveillance"]
+    steward == supports due diligence ==> manufacturer
+    steward == supports ==> foss
```

Comment:
```suggestion
    steward == sustains ==> foss
```
To avoid using the same word twice, I personally prefer "sustains". It summarizes "supports on a sustained basis" and connects with the goal of "ensuring the viability".